Solutions/GoogleWorkspaceReports/Hunting Queries/GWorkspaceSuspendUser.yaml

```diff
@@ -0,0 +1,23 @@
+id: 26cea052-5da9-11ec-bf63-0242ac130002
+name: GWorkspace - Suspend users
+description: |
+  'Query searches suspend users.'

```

Comment:
Wondering if it should be suspend vs suspended users.

Reply:
fixed name/description text